recOrder/plugin/widget/main_widget.py

```diff
@@ -2112,6 +2112,17 @@ def enter_acq_bg_path(self):
         else:
             self.ui.le_bg_path.setText('Path Does Not Exist')
 
+    @Slot(str)
+    def handle_bg_path_update(self,value):
```

Comment:
Consider PEP 8 issue: `(self,value)` -> `(self, value)`.

Reply:
Also please include docstrings for your public methods.

Reply:
Thank you! I'll be mindful for future PRs.